### PR TITLE
add another note re. Share service deprecation

### PR DESCRIPTION
**Pull request by mtbc**
Reflects https://github.com/openmicroscopy/openmicroscopy/commit/9759e0c5cbaca1f32fbedbe0f3a498a135acc840.

Staged at https://www.openmicroscopy.org/site/support/omero5.3-staging/developers/Modules/Api.html#service-level-1-direct-database-and-hibernate-connections.

### CODE REVIEW

**Comment by jburel** [comment]
ROIService is also deprecated

File: omero/developers/Modules/Api.txt [hunk 1]
```diff
@@ -73,15 +73,15 @@ Service Level 1 (direct database and Hibernate connections)
...
+   working with ROIs. (now deprecated)
 -  ScriptService
    :javadoc:`API <slice2html/omero/api/IScript.html>`
    for uploading and launching Python scripts.
 -  SessionService
    :javadoc:`API <slice2html/omero/api/ISession.html>`
    for creating and working with OMERO sessions.
 -  ShareService
-   :javadoc:`API <slice2html/omero/api/IShare.html>`
+   :javadoc:`API <slice2html/omero/api/IShare.html>` (now deprecated)
```
Comment (hflynn) [inline omero/developers/Modules/Api.txt]: No full stop at the end of this point, the rest all have them

File: omero/developers/Modules/Api.txt [hunk 2]
```diff
@@ -73,15 +73,15 @@ Service Level 1 (direct database and Hibernate connections)
    disk space stats.
 -  RoiService
    :javadoc:`API <slice2html/omero/api/IRoi.html>`
-   working with ROIs.
+   working with ROIs. (now deprecated)
```
Comment (hflynn) [inline omero/developers/Modules/Api.txt]: I would move the full stop to after the (now deprecated)

**Comment by hflynn** [comment]
Couple of very minor nitpicks, once they are sorted, I'm happy for this to be merged if you're happy too @jburel

File: omero/developers/Modules/Api.txt (hunk 1 above)
Comment (mtbc) [reply omero/developers/Modules/Api.txt]: Ah, I figured they were associated with the explanatory text that followed (cf. ProjectionService).

**Comment by jburel** [comment]
few comments: 
*  ``(switching groups etc).`` should be `` (switching groups etc.).``
* take opportunity to add description to ProjectionService i.e. `` Provides methods for performing projections of Pixels sets.``
* Update service: remove ``delete`` from the description since it is now deprecated and not the recommended way to delete objects
* ``OMERO.insight works almost exclusively with the IContainer interface`` maybe update to indicate that it mainly uses the Java Gateway cc @dominikl Link to the page

**Comment by mtbc** [comment]
Huh, my --depends-on didn't work?

**Comment by jburel** [comment]
Maybe remove the link, we merge this PR and @dominikl will add the link in his PR

**Comment by mtbc** [comment]
Okay: @dominikl please feel free to cherry-pick https://github.com/openmicroscopy/ome-documentation/commit/609b522cc2d6be196f1a6c8398aa38bcb3cae0d8

**Comment by dominikl** [comment]
Thanks @mtbc . Couldn't cherry-pick for some reason, but basically just copy/pasted your commit to ce927c9590f.

**Comment by mtbc** [comment]
Force-pushed last commit to remove conflicts with @dominikl's latest.

**Comment by jburel** [comment]
looks good thanks

**Pull request merged**